examples/lie_labs.py

```diff
@@ -1,7 +1,7 @@
 import torch
```

Comment:
Do need rename this file? Maybe `trochlie` instead of `lie_labs`?

Reply:
Yes, but after we merge all of this stuff, I have to fix the library import to `torchlie` in multiple places, so we can do that then.